Code/GraphMol/molopstest.cpp

```diff
@@ -7466,6 +7466,41 @@ void testGithub1928() {
   BOOST_LOG(rdInfoLog) << "Finished" << std::endl;
 }
 
+void testGithub1990() {
+  BOOST_LOG(rdInfoLog) << "-----------------------\n Testing Github issue "
+                          "1990: removeHs screws up bond stereo"
+                       << std::endl;
+  {
+    std::unique_ptr<RWMol> mol(SmilesToMol("F/C=C/F"));
+    TEST_ASSERT(mol);
+    MolOps::addHs(*mol);
+    MolOps::removeHs(*mol);
+    TEST_ASSERT(mol->getNumAtoms() == 4);
+    TEST_ASSERT(mol->getBondWithIdx(1)->getStereoAtoms().size() == 2);
+  }
+  {  // make sure that stereo is removed when it comes from Hs:
```

Comment:
Shouldn't this be "stereo is NOT removed"?

Reply:
? I’m not sure what you’re asking

Reply:
The PR is for retaining stereo, the comment is stating we ensure stereo is removed, but looking at the examples stereo is retained, so I'm obviously missing something or just confused.

Reply:
Unless you mean, remove the H atoms from the list of stereo atoms...

Reply:
Ah. I was being stupid. Sorry.
Yes, the comment is misleading. I will fix it.